config.yaml

```diff
@@ -6,10 +6,10 @@ inflation_genesis:
   mint_denom: acudos
   gravity_account_address: cudos16n3lc7cywa68mg50qhp847034w88pntq8823tx
 apr_genesis:
-  initial_height: 1069391
-  norm_time_passed: 0.701184759708210690
-  real_blocks_per_day: 14048
-  blocks_per_day: 12345
+  initial_height: 2704699
+  norm_time_passed: 1.06390993412731
+  real_blocks_per_day: 13824
+  blocks_per_day: 13824
```

Comment:
It might just be a minor change in [this function](https://github.com/CudoVentures/cudos-stats-v2-service/blob/2750dd5302f28d6923c2400de85b057bcc816d0c/internal/tasks/tasks.go#L107), but since `blocks_per_day` and `real_blocks_per_day` are now the same and we are aiming for it to stay that way, should we get rid of the redundant variable?

Reply:
It may be better to keep it in case average block time changes significantly again and we end up having to do another adjustment like last time. What do you think?

Reply:
Agreed, let's leave it in for now, it's easier to change this than to do the gov proposal..!
If we ever get round to it, we can also tidy up more things in one go in a future refactor,